opm/autodiff/BlackoilDetails.hpp

```diff
@@ -276,7 +275,7 @@ namespace detail {
                 for ( int idx = 0; idx < np; ++idx )
                 {
                     B_avg[idx] = B.col(idx).sum()/nc;
-                    maxCoeff[idx] = tempV.col(idx).maxCoeff();
+                    //maxCoeff[idx] = tempV.col(idx).maxCoeff();
```

Comment:
This looks wrong to me and may also partially explain "quicker convergence" since this disables one of the convergence tests (I think).